translations/messages.bg.yaml

```diff
@@ -5,6 +5,8 @@ Add track : Добави следа
 Added : Добавена на
 Climb : Изкачване
 Descent : Спускане
+Description in Bulgarian: Описание на Български
+Description in English: Описание на Английски
```

Comment:
Български and Английски should be lowercased. It's an object, not the country.

Reply:
I checked this https://www.grammarly.com/blog/capitalization-countries-nationalities-languages/

You should capitalize the names of countries, nationalities, and **languages** because they are proper nouns—English nouns that are always capitalized.
> English is made up of many languages, including Latin, German, and French.

I think it should be uppercase

Reply:
nvm, fixed